engine/src/mac-av-player.mm

```diff
@@ -845,7 +845,7 @@ - (void)setPlayer:(AVPlayer *)player
     // PM-2014-07-15 Various tweaks to handle all cases of playback
     if (!m_play_selection_only)
     {
-        if (m_finished && CMTimeCompare(m_player . currentTime, m_player . currentItem . duration) >= 0)
+        if (m_finished && CMTimeCompare(m_player . currentTime, m_player . currentItem . asset . duration) >= 0 && rate > 0)
```

Comment:
What's the difference between using the AVAsset.duration instead of the AVPlayerItem.duration?